### DISCUSSION TITLE
Is there a way to import .bib files _without_ pinning the original key?

### DISCUSSION THREAD

**Discussion by stadlerb** [retorquere/zotero-better-bibtex#2756] [Q&A] [answered]
I often import .bib files from various conference proceedings or journal websites when there's no DOI available, and don't want to keep the original bibtex keys (but I do want to pin some keys, possibly with manual changes). I often forget to unpin the keys immediately after importing, so I have to scroll through all my entries from time to time to un-pin keys I don't want pinned, but I have to take care not to un-pin those keys that I pinned manually.

Is there a way to import from clipboard without pinning the keys, or to disable pinning on import altogether?

**Comment by retorquere** [comment, marked as answer]
Yes: https://retorque.re/zotero-better-bibtex/installation/preferences/hidden-preferences/#importcitationkey

**Reply by stadlerb** [reply]
It works, thanks for the quick response!
